pyEpiabm/pyEpiabm/core/household.py

```diff
@@ -65,3 +67,31 @@ def add_person(self, person):
         """
         self.persons.append(person)
         person.household = self
+        if person.infection_status == InfectionStatus.Susceptible:
+            self.add_susceptible_person(person)
+
+    def add_susceptible_person(self, susceptible_person):
+        """Adds a person to the list of susceptible people in the household.
+
+        Parameters
+        ----------
+        susceptible_person : Person
+            Person to be added
+
+        """
+        if susceptible_person not in self.susceptible_persons:
+            self.susceptible_persons.append(susceptible_person)
+            susceptible_person.household = self
+
+    def remove_susceptible_person(self, non_susceptible_person):
+        """Removes an susceptible person from the list of susceptible people
+        in the household.
+
+        Parameters
+        ----------
+        non_susceptible_person : Person
+            Person to be removed
+
+        """
+        self.susceptible_persons.remove(non_susceptible_person)
+        non_susceptible_person.household = self
```

Comment:
Just trying to understand this line, so it sets their household to be this one as well? Is it possible to be in self.susceptible_persons but not have person.household = self?

Reply:
Yes it seems like their household should already be set here!

Reply:
See update, sorted now